tests/acceptance/test_dashboard.py

```diff
@@ -74,3 +74,28 @@ def test_new_dashboard(self):
             self.browser.get(self.path)
             self.browser.wait_until_not('.loading-indicator')
             self.browser.snapshot('new dashboard')
+
+
+class EmptyDashboardTest(AcceptanceTestCase):
+    def setUp(self):
+        super(EmptyDashboardTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=None,
+        )
+        self.team = self.create_team(organization=self.org, name='Mariachi Band')
```

Comment:
Where is the project `bengal` coming from?

Reply:
Where do you see `bengal`? This test case shouldn't have any project

Reply:
Oh, that project should exist in the other test case in this file though

Reply:
Ah oops